api/src/game.ts

```diff
@@ -31,7 +31,7 @@ export const createNewGameState = (): GameState => {
 
   gameState.graph = graph;
   gameState.players = players;
-  gameState.currentPlayer = players[0];
+  gameState.currentPlayerId = players[0].playerNumber;
```

Comment:
To improve readability, try to use the same language between resources.
`players[0].playerNumber` is an ID ? So call it `players[0].id`
Or maybe `gameState.currentPlayerId` refer just to the `playerNumber` ? so call it `gameState.currentPlayerNumber`